### PR TITLE
doc how to transfer repos to the communtiy:

### PR DESCRIPTION
**Pull request by majormoses**
- plugin focused but much applies even if it is not a plugin.
- providing a very high level overview of the things done after transferring and the expectations on when it is considered `closed`

This was discussed during the summit that this was not obvious that this process was in a PR template. This also exposes a few insights into some processes that are still being finalized and will be open sourced soon.

Signed-off-by: Ben Abrams <me@benabrams.it>

### CODE REVIEW

**Comment by mbbroberg** [comment]
I'll review this later today 👍

**Review by mbbroberg — APPROVED** [review]
LGTM. You're welcome to squash it.

File: TRANSFERRING_REPOS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+   - [Transfer a personal repo](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account)
+   - [Transfer a org repo](https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/)
+1. Once the transfer has been initiated to the maintainer he will then transfer it to their account and then to the org (unless they happen to be an owner in both but they would probably not need this guide)
+1. Once this has been completed the maintainer should then mark the issue with a label of `Repo Created` and then do the required setup to do various things like:
+   - terraform setup
+   - travis setup
+   - rubygems setup
+   - badges
+   - cleanup
+1. Once everything is setup the issue should be closed and any converns like adding additional documentation, testing, etc should create issues in the respective repo.
```
Comment (eheydrick) [inline TRANSFERRING_REPOS.md]: concerns.

Comment (majormoses) [reply TRANSFERRING_REPOS.md]: good catch.

**Pull request merged**